DietPi-Config | RPi: Disable VCSM if less than 32 MiB GPU memory are applied

MichaIng opened the pull request:
**Status**: Ready

**ToDo**:
- [x] Apply on DietPi-Update
- [x] Changelog

**Commit list/description**:
+ DietPi-Config | RPi: Disable VCSM if less than 32 MiB GPU memory are applied, since in this case the cut-down firmware is loaded which does not provide the features required for VCSM to succeed. This may reduce boot overhead and memory usage marginally but especially mutes 5 related kernel error messages at boot.

MichaIng committed:
v6.34

+ DietPi-Config | RPi: Disable VCSM if less than 32 MiB GPU memory are applied, since in this case the cut-down firmware is loaded which does not provide the features required for VCSM to succeed. This may reduce boot overhead and memory usage marginally but especially mutes 5 related kernel error messages at boot.

MichaIng committed:
v6.34

+ DietPi-Patch | RPi: Re-apply GPU memory split to disable VCSM if GPU memory is lower than 32 MiB

MichaIng committed:
v6.34

+ CHANGELOG | DietPi-Config: On RPi, when applying less than 32 MiB GPU memory, the VCSM (VideoCore Shared Memory) device and kernel module are disabled now.

MichaIng commented:
Hmm, actually the dtoverlay does not need to be recompiled on every kernel upgrade. The motivation to skip it (only create it once) also is that it is done 4 times due to the 4 kernel architectures on RPi (ARMv6, ARMv7, RPi4-ARMv7, ARMv8).

But I am not sure if it is possible that it needs to be re-compiled in case of a certain change and more importantly what would happen if an invalid dtoverlay is loaded. If this could break boot (in this particular case), then this would be a very good argument to keep rebuilding it.

🤔 Probably a question for the Raspberry Pi forum: https://www.raspberrypi.org/forums/viewtopic.php?t=290607
________
It is btw not easy to test since the device tree compiler fails in the first place if the device path or syntax is wrong. So the issue would be only if a change has been made that renders a path or syntax invalid after after a kernel upgrade.

Joulinar commented on dietpi/func/dietpi-set_hardware:
```diff
@@ -25,7 +25,7 @@ $FP_SCRIPT	soundcard		target_card (non-matching name for reset to default). Add
 $FP_SCRIPT	lcdpanel		target_panel (none to remove all)
 $FP_SCRIPT	headless		enable/disable
-$FP_SCRIPT	gpumemsplit		64/128/256 # RPi only
+$FP_SCRIPT	gpumemsplit		64/128/256... Range: 16-944, RPi only
```
is his missing the `#`  before `RPi only`

MichaIng replied:
Since this is an informational string only (nothing is executed) it doesn't matter here. Probably I'll rework the whole thing by times to align with other help/usage print standards, like here: https://github.com/MichaIng/DietPi/blob/master/dietpi/dietpi-services#L15-L31

Joulinar replied:
ok clear

MichaIng committed:
v6.34

+ DietPi-PREP |  RPi: Apply GPU memory split to have the dtoverlay created which mutes VCSM related error messages when less than 32 MiB GPU memory are applied
